airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/migrations/AirbyteMessageMigrationV0.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol.migrations;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.version.AirbyteVersion;
+import io.airbyte.protocol.models.AirbyteMessage;
+
+/**
+ * Demo migration to illustrate the template. This should be deleted once we added the v0 to v1
+ * migration.
+ */
+// NOTE, to actually wire this migration, uncomment the annotation
+// @Singleton
```

Comment:
Should this be uncommented as part of this PR or do you plan on doing that in a follow on PR?

Reply:
This should not be uncommented, technically this is a migration we do not want to perform, the "unversioned" objects should disappear. I left this class as an example for the time being.